### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 8

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#184] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252669-be4fbcb7-1123-43a8-9da8-77e38a700ed0.png)

**Comment by danieldinella** [comment]
Per il PIE: 22 + 15 + 8 - 13 - 6 - 4 = 22

**Comment by Exyss** [comment]
@danieldinella ti sei dimenticato di re-includere l'intersezione MareMontagnaCollina, altrimenti non è un PIE completo

$22 + 15 + 8 - 13 - 6 - 4 + 4 = 26$

**Comment by danieldinella** [comment]
> @danieldinella occhio che ti sei dimenticato di re-includere l'intersezione MareMontagnaCollina, altrimenti non è un PIE completo
> 
> 22 + 15 + 8 - 13 - 6 - 4 + 4 = 26

Pensavo non andasse aggiunto perché diceva mare O montagna O collina.

**Comment by Exyss** [comment]
> > @danieldinella occhio che ti sei dimenticato di re-includere l'intersezione MareMontagnaCollina, altrimenti non è un PIE completo
> > 22 + 15 + 8 - 13 - 6 - 4 + 4 = 26
> 
> Pensavo non andasse aggiunto perché diceva mare O montagna O collina.

Proprio per questo va incluso, visto che il "o" implica una congiunzione tra insiemi, dunque $(Mare \cup Montagna \cup Collina)$, quindi si tratta di un PIE "standard"
